Tests/HttpSwiftTests/HttpSwiftTests.swift

```diff
@@ -286,6 +286,35 @@ class HttpSwiftTests: XCTestCase {
         XCTAssertEqual(try? socket.readLine(), "Securely connected")
     }
     
+    func testReplaceExistingRoute() {
+        let firstResponseString = "FirstResponseString"
+        let secondResponseString = "FirstResponseString"
```

Comment:
I believe you meant strings to be different
```suggestion
        let secondResponseString = "SecondResponseString"
```